src/js/renderables/ui/SideMenuContainer.js

```diff
@@ -1,6 +1,8 @@
 import * as me from 'melonjs';
 import TowerButton from '../../buttons/TowerButton';
 import PauseButton from '../../buttons/PauseButton';
+import DefeatButton from '../../buttons/DefeatButton';
+import VictoryButton from '../../buttons/VictoryButton';
```

Comment:
Can you remove the buttons from SideMenuContainer?
It wouldn't make sense to put this in a production branch. You can add the functionality to switch to the screens on what makes sense (e.g. enemy or trash can class for game over when lives are 0)